### DISCUSSION TITLE
Additional fields “break” the user card via picture HTML tag

### DISCUSSION THREAD

**Discussion by Falcion** [anuraghazra/github-readme-stats#3790] [Q&A] [answered]
### Describe the bug

When I'm trying to to apply this URL in `<picture>` tag with the template from the README:
```
https://github-readme-stats.vercel.app/api?username=Falcion&show=reviews,discussions_started,discussions_answered,prs_merged,prs_merged_percentage&show_icons=true&theme=dark&custom_title=Recent%20activity%20(Falcion):&text_bold=true&card_width=500&hide_border=true
```

It shows me (no matter what media set is), this "default" card:

![image](https://github.com/anuraghazra/github-readme-stats/assets/57592842/dcdb6d18-24b5-48dd-8285-05433fde7bc8)

But if I use this same URL within "Markdown image": `![]()`, it shows me this:

![image](https://github.com/anuraghazra/github-readme-stats/assets/57592842/f9059c80-3825-437a-89d4-6fec7c4ef5f7)

VS Code also previews card correctly, but Github .MD is not working.

### Expected behavior

Show correct card correlated with provided URL (image 2):

![image](https://github.com/anuraghazra/github-readme-stats/assets/57592842/f9059c80-3825-437a-89d4-6fec7c4ef5f7)

### Screenshots / Live demo link

Screenshots provided in bug description.

**Comment by qwerty541** [comment, marked as answer]
Hey @Falcion. You have to URL encode the query string values when using `<picture>` tag. Here is an example:

```html
<picture>
  <source
    srcset="https://github-readme-stats.vercel.app/api?username=Falcion&show=reviews%2Cdiscussions_started%2Cdiscussions_answered%2Cprs_merged%2Cprs_merged_percentage&show_icons=true&theme=dark&custom_title=Recent%20activity%20(Falcion):&text_bold=true&card_width=500&hide_border=true"
    media="(prefers-color-scheme: dark)"
  />
  <source
    srcset="https://github-readme-stats.vercel.app/api?username=Falcion&show=reviews%2Cdiscussions_started%2Cdiscussions_answered%2Cprs_merged%2Cprs_merged_percentage&show_icons=true&custom_title=Recent%20activity%20(Falcion):&text_bold=true&card_width=500&hide_border=true"
    media="(prefers-color-scheme: light), (prefers-color-scheme: no-preference)"
  />
  <img src="https://github-readme-stats.vercel.app/api?username=Falcion&show=reviews%2Cdiscussions_started%2Cdiscussions_answered%2Cprs_merged%2Cprs_merged_percentage&show_icons=true&custom_title=Recent%20activity%20(Falcion):&text_bold=true&card_width=500&hide_border=true" />
</picture>
```

<picture>
  <source
    srcset="https://github-readme-stats.vercel.app/api?username=Falcion&show=reviews%2Cdiscussions_started%2Cdiscussions_answered%2Cprs_merged%2Cprs_merged_percentage&show_icons=true&theme=dark&custom_title=Recent%20activity%20(Falcion):&text_bold=true&card_width=500&hide_border=true"
    media="(prefers-color-scheme: dark)"
  />
  <source
    srcset="https://github-readme-stats.vercel.app/api?username=Falcion&show=reviews%2Cdiscussions_started%2Cdiscussions_answered%2Cprs_merged%2Cprs_merged_percentage&show_icons=true&custom_title=Recent%20activity%20(Falcion):&text_bold=true&card_width=500&hide_border=true"
    media="(prefers-color-scheme: light), (prefers-color-scheme: no-preference)"
  />
  <img src="https://github-readme-stats.vercel.app/api?username=Falcion&show=reviews%2Cdiscussions_started%2Cdiscussions_answered%2Cprs_merged%2Cprs_merged_percentage&show_icons=true&custom_title=Recent%20activity%20(Falcion):&text_bold=true&card_width=500&hide_border=true" />
</picture>

**Reply by Falcion** [reply]
Issue resolved: thank you very much!

**Reply by qwerty541** [reply]
Glad to help! Feel free to contact me if you need any further assistance.
